### PR TITLE
QAE-879 - E2E Test case for transparent header background overlay and site title color option

### PR DESCRIPTION
**Pull request by SimranChougule1**
### Description

Header-builder → Transparent Header → Design → Background overlay and site title color

### Screenshots

https://share.getcloudapp.com/BluozdE0

### How has this been tested?

npm run test:e2e:interactive -- tests/e2e/specs/customizer/header-builder/transparent-header/background-and-site-title-color.test.js

### Checklist:
- [ ] My code is tested
- [ ] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: tests/e2e/specs/customizer/header-builder/transparent-header/background-and-site-title-color.test.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+				desktop: 'rgb(92, 11, 130)',
+				tablet: 'rgb(101, 37, 7)',
+				mobile: 'rgb(42, 78, 6)',
+			},
+		};
+		await setCustomize( siteTitleColor );
+		await page.goto( createURL( '/' ), {
+			waitUntil: 'networkidle0',
+		} );
+		await page.waitForSelector( '.ast-theme-transparent-header .site-title a' );
```
Comment (bsfsnehal) [inline tests/e2e/specs/customizer/header-builder/transparent-header/background-and-site-title-color.test.js:51]: @SimranChougule1 please check this test case properly running at your local-end,
It showing error.

Comment (SimranChougule1) [reply tests/e2e/specs/customizer/header-builder/transparent-header/background-and-site-title-color.test.js:53]: Resolved

**Comment by bsfsnehal** [comment]
@SimranChougule1 some checks are failing, please resolve this

**Comment by SimranChougule1** [comment]
@bsfsnehal checks are failing due to accessibility error, on local test case is working proper,  @GauravKhupse checking that problem

**Comment by SimranChougule1** [comment]
closing this PR due to compressed size error

**Pull request closed without merging**